presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMergeTDigestFunction.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation;
+
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.block.BlockBuilder;
+import com.facebook.presto.spi.type.DoubleType;
+import com.facebook.presto.spi.type.SqlVarbinary;
+import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.spi.type.TypeParameter;
+import com.facebook.presto.tdigest.TDigest;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.function.BiFunction;
+
+import static com.facebook.presto.spi.type.TDigestParametricType.TDIGEST;
+import static com.facebook.presto.tdigest.TDigest.createTDigest;
+import static io.airlift.slice.Slices.wrappedBuffer;
+import static java.util.Objects.requireNonNull;
+
+public class TestMergeTDigestFunction
+        extends TestMergeStatisticalDigestFunction
+{
+    static final BiFunction<Object, Object, Boolean> TDIGEST_EQUALITY = (actualBinary, expectedBinary) -> {
```

Comment:
I know this is based on existing code, but it would be nice to have a richer comparison, including testing that a series of quantiles are equal (+/- some epsilon)